cyber-security/modules/csls_s3_object_logging/locals.tf

```diff
@@ -0,0 +1,25 @@
+locals {
```

Comment:
Could you add comments to demonstrate the output of these transformations. It could make debugging easier.

Reply:
Done - but I'm not sure it's more readable for it. Let me know what you think.

Reply:
It is better but it's still not great. Oh well.